extensions/amp-jwplayer/0.1/amp-jwplayer.js

```diff
@@ -121,9 +156,22 @@ class AmpJWPlayer extends AMP.BaseElement {
     }
     return placeholder;
   }
+  /**
+  *
+  */
+  getContextualVal() {
+    if (this.contentSearch_ === '__CONTEXTUAL__') {
```

Comment:
`this.contentSearch_ === '__CONTEXTUAL__'` or `this.contentContextual_ === true`? What happens if `contextual` is `true` but `search` is already set to something else (or not set at all)?

Reply:
@dYale Adding you here.

My understanding is that someone may want to mock a contextual search using their own provided term. For example, search for ‘dog’ but maintain the contextual search logic.